src/main/kotlin/vannak/tech/BookStockManagement/repositories/CategoryRepository.kt

```diff
@@ -0,0 +1,9 @@
+package vannak.tech.BookStockManagement.repositories
+
+import org.springframework.data.jpa.repository.JpaRepository
+import vannak.tech.BookStockManagement.domain.models.Category
+import java.util.*
+
+interface CategoryRepository: JpaRepository<Category,Int> {
+    fun findById(id: Long): Category?
```

Comment:
No need to have this function in repository, and please extend this interface from CrudRepository